tests/integration/hive_metastore/test_grants.py

```diff
@@ -25,6 +27,8 @@ def test_all_grants_in_databases(runtime_ctx, sql_backend, make_group):
     view_d = runtime_ctx.make_table(schema_name=schema_a.name, view=True, ctas="SELECT id FROM range(10)")
     table_e = runtime_ctx.make_table(schema_name=schema_c.name)
 
+    wait_group_provisioned(ws.groups, group_a, group_b)
```

Comment:
don't expose this function as public api, wait by default

Reply:
Updated; this method is no longer exposed for tests to use.